source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+     --i-distance-matrix core-metrics-results/unweighted_unifrac_distance_matrix.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --m-metadata-column disease_stat \
+     --o-visualization core-metrics-results/unweighted-unifrac-disease-stat-group-significance.qzv \
+     --p-pairwise
+
+   qiime diversity beta-group-significance \
+     --i-distance-matrix core-metrics-results/unweighted_unifrac_distance_matrix.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --m-metadata-column animations_subject \
+     --o-visualization core-metrics-results/unweighted-unifrac-animations-subject-group-significance.qzv \
+     --p-pairwise
+
+.. question::
+   Are the associations between subjects and differences in microbial composition statistically significant? How about body sites? What specific pairs of body sites are significantly different from each other?
+
+Again, none of the continuous sample metadata that we have for this data set are correlated with sample composition, so we won't test for those associations here. If you're interested in performing those tests, you can use the ``qiime metadata distance-matrix`` in combination with ``qiime diversity mantel`` and ``qiime diversity bioenv`` commands.
+
+Finally, ordination is a popular approach for exploring microbial community composition in the context of sample metadata. We can use the `Emperor`_ tool to explore principal coordinates (PCoA) plots in the context of sample metadata. While our ``core-metrics-phylogenetic`` command did already generate some Emperor plots, we want to pass an optional parameter, ``--p-custom-axes``, which is very useful for exploring time series data. The PCoA results that were used in ``core-metrics-phylogeny`` are also available, making it easy to generate new visualizations with Emperor. We will generate Emperor plots for unweighted UniFrac and Bray-Curtis so that the resulting plot will contain axes for principal coordinate 1, principal coordinate 2, and days since the experiment start. We will use that last axis to explore how these samples changed over time.
+
+.. command-block::
+
+   qiime emperor plot \
+     --i-pcoa core-metrics-results/unweighted_unifrac_pcoa_results.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --p-custom-axes animations_gradient \
+     --o-visualization core-metrics-results/unweighted-unifrac-emperor-animations_gradient.qzv
+
+   qiime emperor plot \
+     --i-pcoa core-metrics-results/bray_curtis_pcoa_results.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --p-custom-axes animations_gradient \
+     --o-visualization core-metrics-results/bray-curtis-emperor-animations_gradient.qzv
+
+.. question::
+    Do the Emperor plots support the other beta diversity analyses we've performed here? (Hint: Experiment with coloring points by different metadata.)
+
+.. question::
+    What differences do you observe between the unweighted UniFrac and Bray-Curtis PCoA plots?
+
+Alpha rarefaction plotting
```

Comment:
since alpha rarefaction can help inform even sampling decisions, I recommend moving this ahead of the core-metrics example.

Reply:
It might be cool to move this section up sooner, since many folks will consider this plot when picking a rarefaction depth.